akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -457,6 +457,10 @@ trait FlowOps[+Out, +Mat] {
    */
   def filter(p: Out ⇒ Boolean): Repr[Out, Mat] = andThen(Filter(p.asInstanceOf[Any ⇒ Boolean]))
 
+  def takeWhile(p: Out ⇒ Boolean): Repr[Out, Mat] = andThen(TakeWhile(p.asInstanceOf[Any ⇒ Boolean]))
+
+  def dropWhile(p: Out ⇒ Boolean): Repr[Out, Mat] = andThen(DropWhile(p.asInstanceOf[Any ⇒ Boolean]))
+
```

Comment:
OK, let's add docs and Java API (remember that we also have tables of ops in the docs).